lib/goodcity/organisation_populator.rb

```diff
@@ -0,0 +1,57 @@
+module Goodcity
```

Comment:
@trojanh I'd like to retire the lib/classes folder and switch to using the lib/goodcity namespace so I've moved this class here.